lisa/node.py

```diff
@@ -1186,6 +1192,9 @@ class NodeHookSpec:
     def get_node_information(self, node: Node) -> Dict[str, str]:
         ...
 
+    @hookspec
```

Comment:
If it's only hooked in LISA upstream code, it doesn't need to use the hookspec. Just implement as a normal method.